Resources/Storage/GFAL2_StorageBase.py

```diff
@@ -710,6 +713,9 @@ def __getSingleMetadata( self, path ):
           metadataDict['Unavailable'] = 1
         else:
           metadataDict['Unavailable'] = 0
+        if attributeDict['user.status'] == 'ONLINE_AND_NEARLINE':
```

Comment:
Yes but what you have wont work, since for xroot it should return 'Cached' to 1, and 0 for the rest. Unless 'ONLINE' is also a status returned by gfal2 when querying xroot, which was my other question ?